gear-test/src/check.rs

```diff
@@ -562,7 +575,8 @@ where
                         let mut progs: Vec<Program> = final_state
                             .actors
                             .into_iter()
-                            .map(|(_, v)| v.program)
+                            .filter_map(|(_, v)| v)
+                            .map(|v| v.program)
```

Comment:
First of all, this is more beauty way:
```suggestion
                            .filter_map(|(_, actor_opt)| actor_opt.map(|v| v.program))
```
Secondly, better to push an error in case of trying to check memory of terminated program

Reply:
there is a second place with the same code, should be resolved too

Reply:
done